app/javascript/src/components/_token.scss

```diff
@@ -9,7 +9,7 @@
     absolute;
 
   --base-border: 2px;
-  --base-size: 36px;
+  --base-size: 42px;
```

Comment:
If we’re setting this, and then migrating our maps at 43, is that going to make the tokens in our campaign too close to the grid size, which you were trying to avoid? I may be misunderstanding though.

Reply:
The migration will set the grid size of existing maps to 43px. With this MR, maps are now scaled to a grid size of  50px when rendered at zoom scale 0. So our 43px maps will be slightly enlarged. Does that explain it clearly?

Reply:
Yup, I see that now. Thanks.